content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,623 @@
+
+![Single Handler with Single Non-Matching Contact](/images/contact-routing/single-handler/single-non-matching-contact.png)
+
+### Single Handler with Multiple Matching Contacts
+
+In this configuration we have a single handler, **email**, and multiple contacts, **support** and **dev**, configured. The support and dev contacts have a configuration override for **email** handler to send their emails to "support@sensuapp.io" and "dev@sensuapp.io" respectively.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email"
+      ],
+      "contacts": [
+        "support",
+        "dev"
+      ]
+    }
+  },
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.io"
+      }
+    },
+    "dev": {
+      "email": {
+        "to": "dev@sensuapp.io"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+In this instance, although we are using only one handler **email**, the handler is ran multiple times to send an email one to "support@sensuapp.io" and one to "dev@sensuapp.io":
+
+![Single Handler with Multiple Matching Contacts](/images/contact-routing/single-handler/multiple-matching-contacts.png)
+
+### Single Handler with Multiple Non-Matching Contacts
+
+For this configuration we have a single handler **email** and multiple contacts, **support** and **dev** configured. The "support" contact has an override configuration for **slack** and the **dev** contact has an override configuration for **pagerduty**, neither of which have an override configuration for **email**.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email"
+      ],
+      "contacts": [
+        "support",
+        "dev"
+      ]
+    }
+  },
+  "contacts": {
+    "support": {
+      "slack": {
+        "channel": "#support"
+      }
+    },
+    "dev": {
+      "pagerduty": {
+        "service_key": "foobarkey"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+In this instance, since the contacts defined do not have an override configuration for **email** the default configuration for **email** will be used, in our example an email will be sent to "default@example.com"
+
+![Single Handler with Multiple Non-Matching Contacts](/images/contact-routing/single-handler/multiple-non-matching-contacts.png)
+
+### Single Handler with Some Matching Contacts
+
+In this configuration we have a single handler **email** and multiple contacts **support**, **dev** and **eng** configured. The **support** contact has an override configuration for **email**, the **dev** contact has an override configuration for **pagerduty** and the **eng** contact has an override configuration for **slack**.
+
+Since two contacts 
```

Comment:
Not sure if this was to be continued or need to be removed.

Reply:
removed